recipes-core/images/xenclient-installer-image/grub.cfg

```diff
@@ -2,9 +2,8 @@ set timeout=10
 
 menuentry --hotkey=i 'OpenXT Install' {
     set background_color=black
-    multiboot2 /isolinux/xen.gz placeholder flask=disabled console=com1 com1=115200,8n1,pci mbi-video vga=current loglvl=debug guest_loglvl=debug sync_console
-    module2    /isolinux/vmlinuz placeholder quiet root=/dev/ram rw start_install=new eject_cdrom=1 answerfile=/install/answers/uefi.ans console=/dev/tty2 selinux=0
-    module2    /isolinux/rootfs.gz
+    linux   /isolinux/vmlinuz placeholder quiet root=/dev/ram rw start_install=new eject_cdrom=1 answerfile=/install/answers/uefi.ans console=/dev/tty2 selinux=0
+    initrd  /isolinux/rootfs.gz
```

Comment:
@eric-ch how about the other menu options, should they also be given the same treatment?